### PR TITLE
pdf style improvements

### LINKED ISSUE #881: Export and print for Calculated prod and cons screen
Related to:
- [ ] 18.	For the “Calculated Amounts” window, can we make it printable and exportable? (Calculated production and consumption)

and

Similarly, if the current "Calculated amounts" report can be enhanced, it too can replace our current report that compares production and consumption with the baseline. However this one requires more work to get there. 

A solution to be proposed.

### PR DESCRIPTION
**Pull request by nico4**
Closes #881

### CODE REVIEW

File: ozone/core/api/export_pdf/art7/others.py [hunk 1]
```diff
@@ -25,10 +137,32 @@ def get_prodcons_data(period, parties):
...
+            '',
+            "{label}**".format(label=_('CONSUMPTION')),
+            '',
+            '',
+            '',
+            ''
+        ),
+        (
+            '',
+            '2016',
```
Comment (melish) [inline ozone/core/api/export_pdf/art7/others.py]: don't hardcode 2016, reporting period should be a parameter (submission.reporting_period.name)

File: ozone/core/api/export_pdf/art7/others.py [hunk 2]
```diff
@@ -139,18 +274,21 @@ def get_prodcons_data(period, parties):
...
+                limit_prod,
+                actual_cons,
+                baseline_cons,
+                chng_cons,
+                limit_cons,
+                per_capita_cons
+            )
         data['tables'].append(table_data)
 
     total['actual_prod'] = round(total['actual_prod'], 2)
```
Comment (melish) [inline ozone/core/api/export_pdf/art7/others.py]: Use models.utils.round_half_up instead of round

File: ozone/core/api/export_pdf/util.py [hunk 3]
```diff
@@ -69,7 +71,7 @@ def _style(style_name, **kwargs):
 no_spacing_style = _style('BodyText', alignment=TA_LEFT, fontSize=FONTSIZE_DEFAULT, spaceBefore=0)
 
 
-left_description_style = _style('BodyText', alignment=TA_LEFT, fontSize=FONTSIZE_DEFAULT, spaceBefore=0)
+left_description_style = _style('BodyText', alignment=TA_LEFT, fontSize=FONTSIZE_SMALL, spaceBefore=-1)
```
Comment (melish) [inline ozone/core/api/export_pdf/util.py]: better use a variable name related to the style itself, not to a specific usage

File: ozone/core/api/export_pdf/art7/others.py [hunk 4]
```diff
@@ -8,10 +12,118 @@
     Obligation,
 )
 
+from ..util import p_l
+from ..util import h1_style, h2_style, h3_style, page_title_style, FONTSIZE_SMALL, TABLE_STYLES
+from ..util import left_description_style
+from ..util import col_widths
+
+
+TABLE_CUSTOM_STYLES = (
```
Comment (melish) [inline ozone/core/api/export_pdf/art7/others.py]: TABLE_STYLES from util might overwrite some of your custom settings, like the FONTSIZE.

File: ozone/core/api/export_pdf/art7/others.py (hunk 1 above)
Comment (nico4) [reply ozone/core/api/export_pdf/art7/others.py]: Got it

File: ozone/core/api/export_pdf/art7/others.py (hunk 2 above)
Comment (catalinjitea) [reply ozone/core/api/export_pdf/art7/others.py]: Done.

**Comment by melish** [comment]
Already rebased & merged

**Pull request closed without merging**